### DISCUSSION TITLE
Where can I enable mass edit, import, export for users?

### DISCUSSION THREAD

**Discussion by davinhun** [YetiForceCompany/YetiForceCRM#16882] [How to] [answered]
Hello
As an administrator I can do these activities (mass edit, import, export) , but where can I set to enable it for a users or profiles?

**Comment by apcloic** [comment, marked as answer]
Hello @hunterk43
Like for custom filter creation,  Mass action rights for users are in profiles settings :
https://github.com/YetiForceCompany/YetiForceCRM/issues/3460#issuecomment-243509964
